UnitTests/TextViewTests.cs

```diff
@@ -1930,6 +1929,42 @@ public void LoadStream_LF ()
 			Assert.Equal ($"This is the first line.{Environment.NewLine}This is the second line.{Environment.NewLine}", tv.Text);
 		}
 
+		[Fact]
+		public void LoadStream_IsDirty_AllowsNullTerminator ()
+		{
+			var text = "Testing";
```

Comment:
I don't see any tests that show what happens if a `null` is in the middle of the text. Should there be a tests for that?

Reply:
Yes you are right. I'll do it.